terraform/gcp/inventory.tmpl

```diff
@@ -4,6 +4,9 @@ all:
     use_sbd: ${use_sbd}
     gcp_cluster_ip: ${hana-vip}
     prefix: ${name_prefix}
+    project: ${gcp_project}
+    primary_zone: ${gcp_primary_zone}
```

Comment:
Similar to the other comment, we can think about moving them to hosts>>hana section of the playbook